You can view the build output for this pull request here: https://travis-ci.org/go-amz/amz/jobs/45584945

The tests fail with:
```
ec2/ec2.go:25:2: cannot find package "gopkg.in/amz.v1/aws" in any of...
```

I don't think doing a `go get gopkg.in/amz.v1/...` is the right answer. That would be mixing code from the release version with code from the pull request under test.

In my limited experience with gopkg.in, I think we need to:

* [ ] have the `master` branch use internal imports with `github.com/go-amz/amz`
* [x] have the `v1` branch continue to use internal imports with `gopkg.in/amz.v1`
* [ ] have the `v2-dev` branch use `gopkg.in/amz.v2-dev` (once https://github.com/niemeyer/gopkg/issues/25 is done). currently v2-dev uses v1 code, also mixing code from different versions.
* [ ] Make `master` the default branch in GitHub, the default place pull requests are open to.
* [ ] Before tagging a release, merge over to the `v1` branch and ensure all import paths are correct.
* [ ] Advise users to import v1, but don't disallow use of or contribution to master (this is [what I do for fsnotify](https://github.com/go-fsnotify/fsnotify#api-stability), but my CONTRIBUTING doc could use some work).

If there is a better way to do this, I'd love to hear it (I have a tendency to overcomplicate things sometimes). /cc @niemeyer

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,20 @@
+sudo: false
+language: go
+
+go:
+  - 1.2
+  - 1.4
+
+install:
+ - go get gopkg.in/check.v1
+
+before_script:
+  - export DEFAULT_TRAVIS_BUILD_DIR=$TRAVIS_BUILD_DIR
+  - export TRAVIS_BUILD_DIR=$HOME/gopath/src/gopkg.in/amz.v1
+  - mkdir -p $TRAVIS_BUILD_DIR
+  - mv -vfT $DEFAULT_TRAVIS_BUILD_DIR $TRAVIS_BUILD_DIR
+
+script:
+  - go build ./...
+  - go test -v ./...
+  - FIXED=$(go fmt ./... | wc -l); if [ $FIXED -gt 0 ]; then echo "gofmt - $FIXED file(s) not formatted correctly, please run gofmt to fix this." && exit 1; fi

--- a/README.md
+++ b/README.md
@@ -3,6 +3,8 @@ goamz
 
 Go package to interact with Amazon Web Services
 
+[![Build Status](https://travis-ci.org/go-amz/amz.svg)](https://travis-ci.org/go-amz/amz)
+
 Instructions
 ------------
 

